mParticle-iterable/MPKitIterable.m

```diff
@@ -148,7 +150,15 @@ - (MPKitExecStatus *)onIdentifyComplete:(FilteredMParticleUser *)user request:(F
     return [[MPKitExecStatus alloc] initWithSDKCode:[[self class] kitCode] returnCode:MPKitReturnCodeSuccess];
 }
 
-- (NSString *)getPlaceholderEmail:(FilteredMParticleUser *)user {
+- (NSString *)getPlaceholderEmail:(NSString *)userId {
```

Comment:
All of this implementation for the new functionality was actually totally fine! I basically just included these changes as-is in my PR (#11). The only thing I changed was renaming `id` to `userId`, but that was in our existing code and is just a style improvement.

Why the need to make all the other "fixes" to our existing code? If this PR had just included the new functionality, baring the weird use of a pointer for the BOOL variable, this could have basically been merged without issue...